scripts/scil_filter_tractogram_anatomically.py

```diff
@@ -246,28 +323,42 @@ def main():
         ctx_mask == int(1))] = 1
```

Comment:
Are this comparisons necessary? The arrays contain binary values, so foreground should evaluate to `True` and background to `False` without needing to compare to `1`. Please check that this.

Reply:
Additionally, the backslash is not necessary.

Reply:
Done in dbd0b2a.